LICENSE.txt

```diff
@@ -0,0 +1,5 @@
+the fuwwy amogus wicense
+
+if you are a fuwwy, or identify as a sussy amogus impasta baka, you are permitted to copy, sell and use this work in any manner you choose without need to include any attwibution you do not see fit. 
+you are asked as a couwtesy to wetain this wicense in any derivatives but you are not wequiwed.
+if do not belong the abovementioned categowy, go away.
```

Comment:
uh

Reply:
yes?

Reply:
uh

Reply:
i'm not sure whether this is a good license for this project